azure-quantum/azure/quantum/job/job.py

```diff
@@ -142,21 +142,212 @@ def get_results(self, timeout_secs: float = DEFAULT_TIMEOUT):
 
             if self.details.output_data_format == "microsoft.quantum-results.v1":
                 if "Histogram" not in results:
-                    raise f"\"Histogram\" array was expected to be in the Job results for \"{self.details.output_data_format}\" output format."
+                    raise ValueError(f"\"Histogram\" array was expected to be in the Job results for \"{self.details.output_data_format}\" output format.")
```

Comment:
Since this method now raises a `ValueError`, it would be good to include this infor in its doc comment.